README.md

```diff
@@ -5,7 +5,126 @@
 
 ## Getting started
 
-TODO
+|Package| Download Link                                                                                        | Description                                                                |
+|----|------------------------------------------------------------------------------------------------------|----------------------------------------------------------------------------|
+|Workleap.DomainEventPropagation.Abstractions| ![nuget](https://img.shields.io/nuget/v/Workleap.DomainEventPropagation.Abscratcions.svg?logo=nuget) | Contains abstractions that are used for publishing and recieveing messages |
+|Workleap.DomainEventPropagation.Publishing| ![nuget](https://img.shields.io/nuget/v/Workleap.DomainEventPropagation.Publishing.svg?logo=nuget)   | Contains classes for message publishing                                    |
+|Workleap.DomainEventPropagation.Subscription| ![nuget](https://img.shields.io/nuget/v/Workleap.DomainEventPropagation.Subscription.svg?logo=nuget) |  Contains classes for subscriptions                                        |
+
+### What does the Workleap.DomainEventPropagation package do?
+* Provides list of existing Topics
+* Serializes domain events
+* Manages publication fields
+* Provides webhook endpoint (.net core only)
+* Provides IDomainEvent and IDomainEventHandler<> interfaces
+* Deserializes domain events
+* Mediates call to DomainEventHandlers
+* Validates configurations
+* Handles topic subscriptions
+
+### Using the Workleap.DomainEventPropagation.Publishing package to publish events
+
+When using dotnet core, you can register event propagation at startup in the service collection.
```

Comment:
This is actually not true, we're targeting .NET Standard 2.0 so you can publish events from (almost) any version of .NET.